app/controllers/user_sessions_controller.rb

```diff
@@ -1,9 +1,15 @@
 class UserSessionsController < ApplicationController
-  before_filter :reset_session, only: [:destroy]
-  before_filter :require_no_user, :only => [:new, :create]
-  before_filter :require_user, :only => :destroy
+  before_action :reset_session, only: [:destroy]
+  before_action :require_no_user, only: %w[new create]
+  before_action :require_user, only: :destroy
 
-  def security_notification; end
+  def new
+    construct_user_session
+  end
+
+  def security_notification
+    redirect_to(account_path) if current_user
```

Comment:
Specs?

Reply:
cucumber spec caught it.

Reply:
There is already a spec in user_session_controller. Also cucumber is testing it to redirect to the account_path.